build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.3.41'
```

Comment:
Why you have downgraded this?

Reply:
There was this "Kotlin version is used for building with Gradle (1.3.50) differs from the one bundled into the IDE plugin (1.3.41)"

Reply:
Then that means your kotlin version is outdated, do not downgrade it, instead update yourself